bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/gems/GemsMagnitudeTable.scala

```diff
@@ -23,9 +22,44 @@ import Scalaz._
  * (that is, with predefined magnitude limits).
  */
 object GemsMagnitudeTable extends MagnitudeTable {
+  /**
+    * Normally, each type of condition (CC, IQ, SB) affect magnitude limits:
+    * 1. independently (e.g. the CC's affect on mag limits does not depend on the SB); and
+    * 2. uniformly (e.g. if CC changed the faintness limit, it also changed the saturation limit by the same amount).
+    *
+    * In NGS2, however, this is no longer the case:
+    * 1. The value of one condition can affect another's effect on magnitude limits.
+    * 2. For a given set of conditions, they might affect faintness and saturation limits differently.
+    *
+    * Thus, to create the magnitude limits adjuster for NGS2, we need to use some customization as per the following
+    * functions:
+    */
```

Comment:
I kind of think it might be easier to read and maintain if we just keep a `Map` somewhere that matches the one in the requirements.  It definitely predates this PR but all the adjustments are hard for me to follow.